meteor-app/imports/ui/layouts/full-window/style.less

```diff
@@ -10,6 +10,8 @@
   align-items: stretch;
 
   .page__header {
+    .flex-row;
```

Comment:
The chunk here (line 5 to line 22 previously) described explicitly the flex relationship between the parent and the children. While the line you added here is only relevant to the child itself. It does not belong here.